packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -716,12 +725,14 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     schemaComposer.add(StringScalarInputType);
     schemaComposer.add(BooleanScalarInputType);
     schemaComposer.add(JSONScalarInputType);
+    schemaComposer.add(JSONObjectScalarInputType);
```

Comment:
Do you see dedicated scalar input types as well?

Reply:
These inputs are only added for filtering.